src/writers.ts

```diff
@@ -0,0 +1,54 @@
+import { CheckpointWriters } from '@snapshot-labs/checkpoint';
+import { hexStrArrToStr, toAddress } from './utils';
+
+export const writers: CheckpointWriters = {
```

Comment:
I chose to use this format so that new users can benefit from the type system provided by the `CheckpointWriters` type definition.  At the moment the individual function type `CheckpointWriter` (without the s) is not exported from the Checkpoint library, but once I make an updated release for that, I'll update this to use the `import * as writers from './writers` format you suggested. For now this works for typesafety.